### PR TITLE
update configure-endpoints-gp.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2731
clarified which SYSTEM account to use

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2371

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/configure-endpoints-gp.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ ms.date: 04/24/2018
 
 5. Right-click **Scheduled tasks**, point to **New**, and then click **Immediate task**.
 
-6. In the  **Task** window that opens, go to the **General** tab. Choose the local SYSTEM user account under **Security options**.
+6. In the  **Task** window that opens, go to the **General** tab. Choose the local SYSTEM user account (BUILTIN\SYSTEM) under **Security options**.
```
Comment (nenonix) [inline windows/security/threat-protection/microsoft-defender-atp/configure-endpoints-gp.md]:
```suggestion
6. In the **Task** window that opens, go to the **General** tab. Choose the local SYSTEM user account (BUILTIN\SYSTEM) under **Security options**.
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix ,
your recommendation was applied, thank you!

**Comment by mypil** [comment]
@mjcaparas - Can you please assist to check, approve and merge the content of this PR based on issue #2371?

Thank you.

**Pull request merged**